src/ssl.c

```diff
@@ -134,6 +134,34 @@
 #endif /* !WOLFCRYPT_ONLY || OPENSSL_EXTRA */
 #ifndef WOLFCRYPT_ONLY
 
+#if defined(OPENSSL_EXTRA) && defined(HAVE_ECC)
+const WOLF_EC_NIST_NAME kNistCurves[] = {
+    {STR_SIZEOF("P-192"),   "P-192",   NID_X9_62_prime192v1},
```

Comment:
I like the `STR_SIZEOF` macro use, but perhaps put it here in the .c file and only for this OPENSSL_EXTRA and HAVE_ECC case? Concerned this macro name is too generic and may collide with customer macros.

Reply:
I put the macro definition in `wolfssl/internal.h` so it isn't included in the final build. I don't think there is a possibility of a collision.